File: Java Script - Introduction to Programming/6.2. Drawing on the Console - Exam Problems/04. Arrow.js
function arrow([n]) {
  n = +n
  //top part
  console.log('.'.repeat(n / 2) + '#'.repeat(n) + '.'.repeat(n / 2))
  //under top
  for (i = 0; i < n - 2; i++) {
    console.log('.'.repeat(n / 2) + '#' + '.'.repeat(n - 2) + '#' + '.'.repeat(n / 2))
  }
  // middle part
  console.log('#'.repeat(n / 2 + 1) + '.'.repeat(n - 2) + '#'.repeat(n / 2 + 1))
  // before bottom
  let dot = 0
  for (i = 1; i <= n - 2; i++) {

    console.log('.'.repeat(i) + '#' + '.'.repeat(Math.abs(n * 2 - 5 - dot)) + '#' + '.'.repeat(i))
    dot += 2
  }
  //bottom
  console.log('.'.repeat(n - 1) + '#' + '.'.repeat(n - 1))
}

// arrow(['3'])

File: Pyrhon - Introduction to Programming/6.2. Drawing on the Console - Exam Problems/04. Arrow.py
n = int(input())
#top part
print('.' * (n // 2) + '#' * n + '.' * (n // 2))
#under top
for i in range( n - 2):
    print('.' * (n // 2) + '#' + '.' * (n - 2) + '#' + '.' * (n // 2))
#$middle part
print('#' * (n // 2 + 1) + '.' * (n - 2) + '#' * (n // 2 + 1))
#before bottom
dot = 0
for i in range(n - 2):
    print('.' * (i + 1) + '#' + '.' * abs(n * 2 - 5 - dot) + '#' + '.' * (i + 1))
    dot += 2
#bottom
print('.' * (n - 1) + '#' + '.' * (n - 1))
